public/app/features/dashboard/components/DashboardRow/DashboardRow.tsx

```diff
@@ -100,6 +110,8 @@ export class DashboardRow extends React.Component<DashboardRowProps> {
           type="button"
           data-testid={selectors.components.DashboardRow.title(title)}
           onClick={this.onToggle}
+          // aria-expanded attribute to inform screen readers
```

Comment:
This comment can be removed. `aria-expanded` implicitly tells us it will be used for assistive technologies.

Reply:
@ckbedwell Thank you for your feedback.